payments/tests/factories.py

```diff
@@ -122,6 +125,16 @@ class Meta:
         model = AdditionalProduct
 
 
+class TalpaProductAccountingFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = TalpaProductAccounting
```

Comment:
To avoid having duplicates on the testing db:
```suggestion
    class Meta:
        model = TalpaProductAccounting
        django_get_or_create = ("region", "product_type",)
```